README.adoc

```diff
@@ -111,7 +111,7 @@ The following table shows the compatibility of the plugin version with Develocit
 |===
 |Bamboo Plugin version  | Develocity Maven extension version        | Common Custom User Data Maven extension version  | Minimum supported Develocity version
 |Next version           | 1.20.1                                    | 1.12.4                                           | 2023.4
-|2.0.0                  | 1.20.1                                    | 1.12.4                                           | 2023.4
+|2.0.0                  | 1.20.1                                    | 1.12.4                                           | 2024.1
```

Comment:
2024.1 should be in line with `Next version` (as 1.21 for maven extension once it's done)

Reply:
Yes, you're right. Since 2.0.0 of the Bamboo plugin is already released.

Reply:
FIXED (5b31a547)